cursorless-talon/src/marks/decorated_mark.py

```diff
@@ -153,7 +153,12 @@ def setup_hat_styles_csv(hat_colors: dict[str, str], hat_shapes: dict[str, str])
 def init_hats(hat_colors: dict[str, str], hat_shapes: dict[str, str]):
     setup_hat_styles_csv(hat_colors, hat_shapes)
 
-    vscode_settings_path: Path = actions.user.vscode_settings_path().resolve()
+    vscode_settings_path: Path | None = None
+
+    try:
+        vscode_settings_path = actions.user.vscode_settings_path().resolve()
+    except Exception as ex:
+        print(ex)
```

Comment:
```suggestion
    try:
        vscode_settings_path = actions.user.vscode_settings_path().resolve()
    except Exception as e:
        print(f"Could not resolve VS Code settings path at {vscode_settings_path} (not fatal): {e}")
```

Reply:
Try to avoid just doing raw printing of exception objects, because then you see the exception message in the log without much context / able to debug them.

Reply:
The exception message is already well formed
`2025-01-16 18:00:29.801    IO Couldn't find VSCode's settings JSON. Tried these paths: C:\Users\andre\AppData\Roaming\Code2\User\settings.json, C:\Users\andre\AppData\Roaming\VSCodium\User\settings.json`

Reply:
It's defined here
https://github.com/cursorless-dev/cursorless/blob/0bad81a015c08e2976728df09fe31d8a9105ec84/cursorless-talon/src/apps/vscode_settings.py#L66-L73